README.md

```diff
@@ -1 +1,162 @@
-# Conference-page
\ No newline at end of file
+<a name="readme-top"></a>
+
+  <h3><b>Essohanam Portfolio README Template</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [xGen Conference] <a name="about-project"></a>
+
+> Hello , This is my first project and its a conference website.
+
+**[CC Environment Global Conference]** "It's a great website with well-defined sections for home and about pages, and it's responsive on screens below 768px as well as on desktop versions." 
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+<details>
+  <summary>HTML</summary>
+</details>
+
+<details>
+  <summary>CSS</summary>
+</details>
+<details>
+  <summary>JavaScript</summary>
+</details>
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Dynamic speakers section]**
+- **[Menu clickable]**
+- **[Hover Buttons]**
+- **[Responsive desplay]**
+- **[colorful texts]**
+- **[About ]**
+- **[Icons ]**
+- **[Validation Form]**
+- **[Aligned items]**
+- **[Desktop screens supported]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+- [Live Demo Link](https://abbassarwar.github.io/XGen-Conference/)
```

Comment:
:clap: 
- [ ] In addition to your live link, I think it will be a lot nicer to add the video presentation link here, as contained in the requirement. :pleading_face:

Reply:
### MISSING VIDEO PRESENTATION

- [ ] As suggested by the previous and as also a key requirement for this project, kindly record a video with the following:

-  Present the features of the project.
- The video's maximum length should not be more than 5 minutes.
- The video contains a demo of the project, showing the pages in different screen sizes.
- The video should contain a highlight of some piece of the code you find interesting and proud of making.

*Check the project requirement to see this missing part of the requirement*